Modules/Services/Sources/Models/Details/BundledRelationValueProvider+CustomFields.swift

```diff
@@ -49,7 +49,9 @@ extension BundledRelationsValueProvider {
         if layoutValue == .note {
             title = snippet
         } else if DetailsLayout.fileLayouts.contains(layoutValue) {
-            title = "\(name).\(fileExt)"
+            let ext = ".\(fileExt)"
+            let fileSuffix = fileExt.isNotEmpty && !name.hasSuffix(ext) ? ext : ""
```

Comment:
We have the same logic in `FileDetails.fileName`. Needs the sync there?

Reply:
Didn't know that, thanks! Unify this logic in FileDetails 🙌

Reply:
If everything is ok, you can merge this PR